action.yaml

```diff
@@ -40,21 +43,23 @@ runs:
         npm install --no-save "@open-turo/commitlint-config-conventional" 2>/dev/null
         cd - &>/dev/null
     - name: Check commitlint config
-      if: hashFiles(inputs.config-file) != '' && inputs.turo-conventional-commit
+      id: turo-commitlint-config
+      if: steps.enable-commit-lint.outputs.commit-lint-enabled == "true" && inputs.turo-conventional-commit == "true"
       shell: bash
       run: |
         # Write default commitlint config
         echo "extends: [\"@open-turo/commitlint-config-conventional\"]" > "${{ inputs.config-file }}"
+        echo '::set-output name=turo-commitlint-config-written::"true"'
     - name: Commitlint
-      if: hashFiles(inputs.config-file) != ''
+      if: steps.enable-commit-lint.outputs.commit-lint-enabled == "true"
       uses: wagoid/commitlint-github-action@v4
       with:
         configFile: ${{ inputs.config-file }}
       env:
         # This /github/home is the internal Docker mount of $RUNNER_TEMP/_github_home
         NODE_PATH: /github/home/node_modules
     - name: Clear commitlint config changes
-      if: env.RESTORE_CONFIG_FILE
+      if: steps.turo-commitlint-config.outputs.turo-commitlint-config-written
```

Comment:
I like this more compact form, eliminating the `== "true"` verbosity. If this form works just fine, what do you think about using this form throughout this and all of our composite Actions?